integrations/adobe-analytics/lib/index.js

```diff
@@ -414,35 +406,43 @@ AdobeAnalytics.prototype.checkoutStarted = function(track) {
 /**
  * Update window variables and then fire Adobe track call
  *
- * @param {*} msg
- * @param {*} adobeEvent
+ * @param {*} msg              Segment Page or Track payload.
+ * @param {string} adobeEvent  Adobe standard event.
  */
 
 AdobeAnalytics.prototype.processEvent = function(msg, adobeEvent) {
-  var props = msg.properties();
-
-  var products = msg.products();
```

Comment:
Just want to call out the removal of `msg.products()`. Facade does some fancy stuff behind the scenes (https://github.com/segmentio/facade/blob/a7cc955f57e14373bb4c502a3ea54a96d8aef9bc/lib/track.js#L382-L386).

In the new changes, it looks like we're calling `properties.products` directly without type checking. This _might_ cause issues if the customer's data has some weird value set for `products`, but I wouldn't be too worried. I just wanted to call it out in case there is a bigger concern.

Reply:
Good callout - thanks!

Reply:
I think this should be ok since we early return if properties.products is not an array: https://github.com/segmentio/analytics.js-integrations/pull/231/files/f7386d3d50e389cbfa9211c2ef754e89dfe768dd#diff-e7435b76f43719e786ff94a43acc1f6eR901